meta-refkit-core/recipes-sensors/mraa/mraa_git.bb

```diff
@@ -4,9 +4,12 @@ AUTHOR = "Brendan Le Foll, Tom Ingleby, Dmitry Rozhkov"
 
 LICENSE = "MIT"
 LIC_FILES_CHKSUM = "file://COPYING;md5=66493d54e65bfc12c7983ff2e884f37f"
+PV = "1.7.0"
 
-SRC_URI = "git://github.com/intel-iot-devkit/mraa.git;protocol=http;tag=v${PV} \
-           "
+SRC_URI = "git://github.com/intel-iot-devkit/${BPN}.git;protocol=http; \
+          "
+
+SRCREV = "${AUTOREV}"
```

Comment:
There's an eSDK bug about AUTOREV usage: https://bugzilla.yoctoproject.org/show_bug.cgi?id=11350. It means we cannot use AUTOREV yet. Let's go with the more traditional way (```PV = "1.7.0-${SRCPV}"``` and ```SRCREV````)